docs/metrics/accelerate.md

```diff
@@ -84,10 +83,10 @@ Pulse uses the average value over a period of time to display aggregate change f
 
 Pulse determines your performance level for this metric as follows:
 
-| Performance level[^1] | Change failure rate  |
-| --------------------- | -------------------- |
-| Elite                 | 0-15%                |
-| Medium                | 15-46%               |
-| Low                   | 46-100%              |
+| Performance level[^1]    | Change failure rate  |
+| ------------------------ | -------------------- |
+| Elite, High, Medium [^2] | 0-15%                |
+| Low                      | 15-100%              |
 
 [^1]: Performance levels are based on the [State of DevOps 2019](https://services.google.com/fh/files/misc/state-of-devops-2019.pdf).
+[^2]: Means are not significantly different based on Tukey's post hoc analysis.
```

Comment:
Do you think it could be worth mentioning the page of the State of DevOps report where this conclusion comes from?

```suggestion
[^2]: Means are not significantly different based on Tukey's post hoc analysis ([State of DevOps 2019](https://services.google.com/fh/files/misc/state-of-devops-2019.pdf) pp. 18).
```